docs/nda.md

```diff
@@ -0,0 +1,70 @@
+NDA Policy
+==========
+
+We, as maintainers of the fwupd and LVFS projects often get asked by companies to sign NDAs and
+we’ve always said no.
+Rather than rewriting a version of this document for each individual request, we’ve summarized the
+key points here.
+
+Before we sign an NDA, we need to spend time to check it, and have a lawyer check it.
+The protection language of an NDA is almost always too broad, vague and ambiguous because usually,
+it’s a “generic NDA” and we need to negotiate to make it more specific, with our lawyer and the
+companies lawyer involved.
+
+That means money and time – which is a really bitter pill to swallow if the company isn’t actually
+contributing financially to fwupd/LVFS.
+The remediation language is almost always open-ended and requires that we agree to eventualities
+that are nearly impossible to define fully.
+We often see language like “irreparable harm” and “cannot adequately be compensated with damages,” etc.
+This means the potential negative outcomes are near limitless
+
+We know it feels as though the protocol used by your device is unique and overflowing with valuable
+intellectual property, but unfortunately it’s rarely so.
+In reality, most firmware update protocols are very similar, and specific device implementation is
+broadly the same as direct competitors.
+
+Firmware updating is a very low level operation, with only a few different ways to achieve the same
+result – so many companies we’ve already worked with (over 100 other vendors!) will have ideas
+similar to whatever you’re trying to keep secret about.
+If another company has a similar protocol to yours, and you discover that they have worked with the
+fwupd team at some point: the fwupd team didn’t tell them your secrets — you probably had the same
+idea at roughly the same time.
+
+At the end of the day, working together in an open source project is a very human business that
+involves networks, relationships and, above all, trust.
+If someone actually lets you down in a confidential situation, they lose all of those.
+That hurts the fwupd/LVFS brand and standing much more than any legal recourse you might have with
+an NDA.
+The LVFS works with all levels of the ecosystem from ISV, to ISV, to ODM to OEM and we rely on
```

Comment:
Is "ISV" meant to appear two times here?

Reply:
Oopa, the second if supposed to be IHV -- thanks.